test/file.test.ts

```diff
@@ -0,0 +1,37 @@
+import { FileTechRadarClient } from "src/services/tech-radar/file";
```

Comment:
To avoid using absolute imports, we can add a configuration on our `tsconfig.json` to enable aliased imposts

```json
"@services/*": ["src/services/*"]
```

and here 
```suggestion
import { FileTechRadarClient } from "@services/tech-radar/file";
```